orion/sites/admin/src/announce/announce.js

```diff
@@ -13,43 +15,27 @@ export class AnnouncePage extends React.Component {
  	}
 
 	render() {
-		var numAnnouncements = 3;
-		let fakeAnnounceA = {
-			date: "02/15/2020",
-			author: "Melon King",
-			message: "Only evil people waste watermelon"
-		};
-		let fakeAnnounceB = {
-			date: "02/16/2020",
-			author: "Fruit Hater",
-			message: "Watermelons are fruit, therefore they deserve to dry on the vine"
-		};
-		/* TODO: will use this to test text wrapping later */
-		let fakeAnnounceC = {
-			date: "02/23/2020",
-			author: "Talkative",
-			message: "This will be a longer message later"
-		};
+		const rows = this.state.list.map ((row, index) => {
+	      return <AnnounceRow key = {index} row = {row}/>
+		});
+		const numRows = rows.length;
+			return (
+				<div id="view-announce">
+					<h1>All Announcements ({numRows}) </h1>
 
-		return (
-			<div id="view-announce">
-				<h1>All Announcements ({numAnnouncements})</h1>
+					<ul className="announce-lists">
+						<li className="li-med">Date</li>
+						<li className="li-med">Author</li>
+						<li className="li-large">Message</li>
+						<li className="li-small"> </li>
+					</ul>
 
-				<ul className="announce-lists">
-					<li className="li-med"> Date </li>
-					<li className="li-med"> Author </li>
-					<li className="li-large"> Message </li>
-					<li className="li-small"> </li>
-				</ul>
-
-				<ul>
-					<AnnounceRow announceObj = {fakeAnnounceA}/>
-					<AnnounceRow announceObj = {fakeAnnounceB}/>
-					<AnnounceRow announceObj = {fakeAnnounceC}/>
-				</ul>
-				<Link to={"/announcements/add"}> <button className="announcement-button"> Add Announcement</button> </Link>
-			</div>
-		);
+					<ul id="view-announce">
```

Comment:
I don't think the id here should also be called `view-announce`. `view-announce` is supposed to be the styles for the overall page in general. We're trying to style a list here.